hs_rest_api/discovery.py

```diff
@@ -13,7 +13,7 @@ class DiscoveryResourceSerializer(HaystackSerializer):
     class Meta:
```

Comment:
I made these changes to fix Scott's API.

Reply:
Will this be used?  I'm not seeing this used in this PR.